Documentation CYF 1.0/pages/api-events.html

```diff
@@ -86,13 +86,24 @@ <h2>Game events</h2><br><br>
             <p>This section is all about game events. Game events are functions in your scripts that the Unitale engine
                 runs at various points in the game. By changing up your behaviour depending on the actions the player
                 takes, you can go beyond a basic encounter and make it great.</p>
+            <!--
```

Comment:
You don't want to say some functions can be called in the Monster scripts if they're not in the Encounter script anymore?